scripts/basic-workflow-tests.sh

```diff
@@ -74,7 +74,7 @@ test_init() {
 
     step Checking config.yaml
     if [ "${MIXTRAL}" -eq 1 ]; then
-        sed -i -e 's/models\/merlinite.*/models\/mixtral-8x7b-instruct-v0\.1\.Q4_K_M\.gguf/' config.yaml
```

Comment:
The default model will now be an absolute path just FYI. This command should still work, but you can drop the `models/` prefix as it's now redundant.